### DISCUSSION TITLE
cloneing a rack fails

### DISCUSSION THREAD

**Discussion by peko-hki** [netbox-community/netbox#7632] [General]
Hi,
I want to create a couple of identical rooms (rack, patch panel , switch, ...). 
First I create one rack completely and exported it. Then I change the floor (from 3 -> 2) and tried to import everything.
I see an error messages: 
"Row 1 rack: Object not found.
  Row 1 face: Cannot select a rack face without assigning a rack. "

But the rack name *IS* correct ...
How can I debug this ? 

Thanks, 
   Peer

**Comment by peko-hki** [comment]
OK, I found a solution. I exported an existing configuration and remove the switches. Then it worked (patchpanel only). 
However there is a fault: exporting a rack config shows "rack_name" will import wants "rack"!

**Reply by candlerb** [reply]
CSV export and import are not compatible with each other in Netbox.  It's known, and I don't think there's any plan to fix.

**Reply by jeremystretch** [reply]
Can't fix what's not broken. :wink:

**Comment by peko-hki** [comment]
OK, good to know ... ;)
